install-devnet-packages.sh

```diff
@@ -0,0 +1,530 @@
+TOTAL_STEPS=10
```

Comment:
I recommand using shabang in first line.
```bash
#!/usr/bin/env bash
```

https://dev.to/meleu/what-the-shebang-really-does-and-why-it-s-so-important-in-your-shell-scripts-2755